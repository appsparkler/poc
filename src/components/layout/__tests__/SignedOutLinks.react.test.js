import React from 'react'
import {BrowserRouter as Router} from 'react-router-dom'
import renderer from 'react-test-renderer'
import SignedOutLinks from '../SignedOutLinks.react'

it('should render correctly', () => {
  expect.assertions(1)
  const component = renderer.create(
      <Router>
        <SignedOutLinks />
      </Router>
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
