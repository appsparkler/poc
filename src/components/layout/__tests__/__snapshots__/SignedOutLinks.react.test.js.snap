// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<ul
  className="right"
>
  <li>
    <a
      href="/signin"
      onClick={[Function]}
    >
      Sign Up
    </a>
  </li>
  <li>
    <a
      href="/signup"
      onClick={[Function]}
    >
      Login
    </a>
  </li>
</ul>
`;
